snappy/DiskCache_Config.py

```diff
@@ -1,5 +1,6 @@
 import os
 import json
+from configUpdate import configUpdate
```

Comment:
This is currently how all internal imports work. It turns out that changing it would require some significant changes. I think that this needs to be filed as a separate issue.